distro.py

```diff
@@ -942,6 +1029,7 @@ def _os_release_info(self):
 
     @staticmethod
     def _parse_os_release_content(lines):
+        # type: (TextIO) -> Dict[str, str]
```

Comment:
Instead of `TextIO` should this be `Iterable[str]`?

Reply:
No, this should remain `TextIO`. This is passed an open file handler. 

The `lines` argument is passed to `shlex` which you can see also requires `TextIO`:

https://github.com/python/typeshed/blob/96e0660fba564d574d9bc69b6b151400d00c7f0a/stdlib/shlex.pyi#L31

Changing this to `Iterable[str]` results in the following failure:

```
distro.py:1047: error: Argument 1 to "shlex" has incompatible type "Iterable[str]"; expected "Union[str, TextIO, None]"
```

Reply:
Sounds good, thanks for the explanation!